### ISSUE TITLE
Additional focus is provided for form controls (checkboxes, radio buttons, text fields etc)

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4632]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 2.4.3 Focus Order

**Test Environment:**
- Test URL: https://clarity.design/documentation/
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0
- Screen reader: NVDA 2020.1

**Steps to reproduce:**
1. Open test url.
2. Go to any form conrol. Let say angular checkboxes section.
3. Using arrow key navigation (up or down arrow keys) try to navigate through the checkbox group.

**Expected result:**
User should be able to navigate through the checkbox group. Each checkbox should have single focus and focusing that checkbox appropriate label, control type and state should be announced. 

**Actual result:**
Additional focus is provided for each of the checkboxes. i.e. First the focus goes on the checkbox and screen reader announces it as unlabeled and then the second focus goes on the visible text label, which is announced by screen reader and screen reader also announces it as clickable.

Similar issue is observed for other components like radio buttons, toggle switch, text areas, text edit fields etc,

**Proposed solution:**
Provide single focus for such form fields.

**Comment by gnomeontherun** [comment]
Hi there, this is an automated message.

In an effort to dedupe and track accessibility reports consistently, this issue is being closed and tracked internally with our Accessibility team. It will still be triaged and addressed as part of ongoing work.
